cmd/spiderpool/cmd/cni_types.go

```diff
@@ -27,10 +27,11 @@ const (
 	CniVersion030 = "0.3.0"
 	CniVersion031 = "0.3.1"
 	CniVersion040 = "0.4.0"
+	CniVersion100 = "1.0.0"
 )
 
 // SupportCNIVersions indicate the CNI version that spiderpool support.
-var SupportCNIVersions = []string{CniVersion030, CniVersion031, CniVersion040}
+var SupportCNIVersions = []string{CniVersion030, CniVersion031, CniVersion040, CniVersion100}
 
 const DefaultLogLevelStr = logutils.LogInfoLevelStr
 
```

Comment:
The code patch looks like a simple addition that adds support for CNI version 1.0.0 to the `SupportCNIVersions` slice. 

There are no apparent bug risks with this patch. However, it may be a good idea to double-check that adding support for version 1.0.0 does not introduce any breaking changes or unexpected behavior.

One potential improvement could be to use a constant or variable to represent CNI version 1.0.0 rather than hardcoding it as a string. This can make it easier to reference and maintain in the future.